package.json

```diff
@@ -38,8 +38,8 @@
     "@testing-library/user-event": "~13.2.1",
     "@types/jest": "~27.0.1",
     "@types/node": "~17.0.23",
-    "@types/react": "~17.0.20",
-    "@types/react-dom": "~17.0.9",
+    "@types/react": "~18.0.5",
```

Comment:
I updated this one due to I have some type issue in my local dev. Not sure it affects to everyone???

Reply:
I remember there is an migration and typing issues like using `createRoot` instead of `render`? If it works well I think go with it

Reply:
not sure, i think we need update in exported web project too

Reply:
yup, note it